src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1906,11 +1909,12 @@ static void glb_update_locked(grpc_exec_ctx *exec_ctx, grpc_lb_policy *policy,
         &glb_policy->lb_channel_on_connectivity_changed, NULL);
   }
 
-  // Propagate update to \a fallback_backend_addresses if a non-empty \a serverlist
+  // Propagate update to \a fallback_backend_addresses if a non-empty \a
+  // serverlist
   // hasn't been received from the balancer.
```

Comment:
Please move this onto the end of the previous line.

Reply:
Done. Should have reviewed every change made by the clang formatter.